core/server/data/meta/asset_url.js

```diff
@@ -39,11 +39,13 @@ function getAssetUrl(path, isAdmin, minify) {
         // we don't want to concat the path with our favicon url
         output += path;
 
-        if (!config.get('assetHash')) {
-            config.set('assetHash', utils.generateAssetHash());
-        }
+        if (config.get('env') === 'production') {
```

Comment:
I would suggest, if this is unnecessary or not related to caching, it shouldn't be part of the PR. If we need an issue, let's raise it 👍